task/hadolint/0.1/samples/run.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: hadolint-pipeline
+spec:
+  params:
+    - name: repository
```

Comment:
I guess `run.yaml` here and the one in `tests/run.yaml` have same contents. Maybe we can remove `samples` directory as they both are identical

Reply:
Fixed